contrib/cirrus/packer/conmon_images.yml

```diff
@@ -25,27 +22,10 @@ variables:
 sensitive-variables:
     - 'GCE_SSH_USERNAME'
     - 'GCP_PROJECT_ID'
-    - 'RHSM_COMMAND'
     - 'SERVICE_ACCOUNT'
 
 # What images to produce in which cloud
 builders:
-    #  v----- is a YAML anchor, allows referencing this object by name (below)
-    - &gce_hosted_image
-      name: 'rhel-7'
-      type: 'googlecompute'
-      image_name: '{{build_name}}{{user `BUILT_IMAGE_SUFFIX`}}'
-      image_family: '{{build_name}}-conmon'
-      source_image: '{{user `RHEL_BASE_IMAGE`}}'
-      disk_size: 20
-      project_id: '{{user `GCP_PROJECT_ID`}}'
-      service_account_email: '{{user `SERVICE_ACCOUNT`}}'
-      communicator: 'ssh'
-      ssh_username: '{{user `GCE_SSH_USERNAME`}}'
-      ssh_pty: 'true'
-      # The only supported zone in Cirrus-CI, as of addition of this comment
-      zone: 'us-central1-f'
-
```

Comment:
Removing the YAML anchor will cause the following YAML alias to break (the alias uses and overrides items from the anchor).  I think something like this will work better:

```diff
diff --git a/contrib/cirrus/packer/conmon_images.yml b/contrib/cirrus/packer/conmon_images.yml
index 7cf918f..d62769a 100644
--- a/contrib/cirrus/packer/conmon_images.yml
+++ b/contrib/cirrus/packer/conmon_images.yml
@@ -30,13 +30,11 @@ sensitive-variables:
 
 # What images to produce in which cloud
 builders:
-    #  v----- is a YAML anchor, allows referencing this object by name (below)
-    - &gce_hosted_image
-      name: 'rhel-7'
+      name: 'fedora-29'
       type: 'googlecompute'
       image_name: '{{build_name}}{{user `BUILT_IMAGE_SUFFIX`}}'
       image_family: '{{build_name}}-conmon'
-      source_image: '{{user `RHEL_BASE_IMAGE`}}'
+      source_image: '{{user `FEDORA_BASE_IMAGE`}}'
       disk_size: 20
       project_id: '{{user `GCP_PROJECT_ID`}}'
       service_account_email: '{{user `SERVICE_ACCOUNT`}}'
@@ -46,11 +44,6 @@ builders:
       # The only supported zone in Cirrus-CI, as of addition of this comment
       zone: 'us-central1-f'
 
-    #      v----- is a YAML alias, allows partial re-use of the anchor object
-    - <<: *gce_hosted_image
-      name: 'fedora-29'
-      source_image: '{{user `FEDORA_BASE_IMAGE`}}'
-
 # The brains of the operation, making actual modifications to the base-image.
 provisioners:
     - type: 'file'
```

Reply:
(edit - added missing `-` in diff above)